routes/web.php

```diff
@@ -7,21 +7,20 @@
 Route::get('/', function () {
```

Comment:
Fix this file. This shouldn't have any modifications since we're only creating APIs.